wwwroot/config.json

```diff
@@ -85,83 +85,117 @@
+            //       "markdownDescription": "stepbystepguide.satelliteimagery.step2.markdownDescription"
+            //     },
+            //     {
+            //       "title": "stepbystepguide.satelliteimagery.step3.title",
+            //     "markdownDescription": "stepbystepguide.satelliteimagery.step3.markdownDescription"
+            //     },
+            //     {
+            //       "title": "stepbystepguide.satelliteimagery.step4.title",
+            //       "markdownDescription": "stepbystepguide.satelliteimagery.step4.markdownDescription"
+            //     }
+            //   ]
+            // },
+            // {
+            //   "title": "stepbystepguide.comparetimeseries.title",
+            //   "footnote": "stepbystepguide.comparetimeseries.footnote",
+            //   "steps": [
+            //     {
+            //       "title": "stepbystepguide.comparetimeseries.step1.title",
+            //       "markdownDescription": "stepbystepguide.comparetimeseries.step1.markdownDescription"
+            //     },
+            //     {
+            //       "title": "stepbystepguide.comparetimeseries.step2.title",
+            //       "markdownDescription": "stepbystepguide.comparetimeseries.step2.markdownDescription"
+            //     },
+            //     {
+            //       "title": "stepbystepguide.comparetimeseries.step3.title",
+            //       "markdownDescription": "stepbystepguide.comparetimeseries.step3.markdownDescription"
+            //     },
+            //     {
+            //       "title": "stepbystepguide.comparetimeseries.step4.title",
+            //       "markdownDescription": "stepbystepguide.comparetimeseries.step4.markdownDescription"
+            //     }
+            //   ]
             // }
-        ],
+          ]
+        },
+        {
+          "icon": "story",
+          "itemName": "storymigration",
+          "markdownText": "## Share & Story Migration\n\n### Guide on updating share links, including stories\n\nIf you've received a warning telling you that a share link was based off an older version of TerriaJS, you'll want to do a few things to recreate them in the latest version of Terria!\n\n#### Creators\n\n1. Identify the share URL which causes a notification of share preservation effort when loading them into Terria. e.g. you may know where they are embedded, or come across them accidentally\n2. Open them in the newer (v8) based map application \n3. Check that everything is sharing as you expect, tweak the layers to fit if some are missing or not behaving correctly.\n4. Update the share link by clicking the Share/Print button and copy the new link\n5. If distribution of share link is known, update the url in your web page (if embedded), or send the new link to users \n\n#### Consumers\n1. Load the share URL\n2. If it doesn't quite look right, notify the provider so they can follow the above steps to recreate it in the latest version of Terria",
```

Comment:
This doesn't seem like it gets parsed correctly, might just be the terriajs version that's on TerriaMap#`next` branch
<img width="800" alt="image" src="https://user-images.githubusercontent.com/9134432/94759946-9f00f900-03e4-11eb-8f74-71f5f89f225d.png">

Reply:
Yep fixed with terriajs 8.0.0-alpha.53 - just waiting for CI now. thanks @na9da 👏